bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/discovery/BridgeDiscoveryParticipant.java

```diff
@@ -145,9 +146,12 @@ public String getServiceType() {
 
         return DiscoveryResultBuilder.create(uid)
                 .withLabel("Bosch Smart Home Controller (" + publicInformation.shcIpAddress + ")")
-                .withProperty("ipAddress", publicInformation.shcIpAddress)
-                .withProperty("shcGeneration", publicInformation.shcGeneration)
-                .withProperty("apiVersions", publicInformation.apiVersions).withTTL(TTL_SECONDS).build();
+                .withProperty(BridgeHandler.THING_PROPERTY_MAC_ADDRESS, publicInformation.macAddress)
+                .withProperty(BridgeHandler.THING_PROPERTY_IP_ADDRESS, publicInformation.shcIpAddress)
+                .withProperty(BridgeHandler.THING_PROPERTY_SHC_GENERATION, publicInformation.shcGeneration)
+                .withProperty(BridgeHandler.THING_PROPERTY_API_VERSIONS,
+                        publicInformation.getApiVersionsAsCommaSeparatedList())
+                .withTTL(TTL_SECONDS).build();
```

Comment:
Regarding https://github.com/openhab/openhab-addons/pull/18391#issuecomment-2724536166, can try to add:
```suggestion
                .withRepresentationProperty(Thing.PROPERTY_MAC_ADDRESS)
                .withTTL(TTL_SECONDS).build();
```

also?

Reply:
Thank you, that did the trick 😎 The discovery result is now not visible, unless I display ignored results 👍